lib/archethic/p2p/message/get_current_replications_attestations.ex

```diff
@@ -0,0 +1,48 @@
+defmodule Archethic.P2P.Message.GetCurrentReplicationsAttestations do
+  @moduledoc """
+  A request to get the replication attestations of current summary
+  The nodes receiving this must be elected to store the given subsets
+  """
+
+  @enforce_keys [:subsets]
+  defstruct [:subsets]
+
+  alias Archethic.Crypto
+  alias Archethic.BeaconChain
+  alias Archethic.P2P.Message.GetCurrentReplicationsAttestationsResponse
+
+  @type t :: %__MODULE__{subsets: list(binary())}
+
+  @spec process(message :: __MODULE__.t(), sender_public_key :: Crypto.key()) ::
+          GetCurrentReplicationsAttestationsResponse.t()
+  def process(%__MODULE__{subsets: subsets}, _) do
+    replications_attestations =
+      Enum.flat_map(subsets, fn subset ->
+        BeaconChain.get_current_summary_replication_attestations(subset)
```

Comment:
You should also concatenate with the current subset slot attestations (`Subset.get_current_slot`) and reduces the confirmations also